va/validation-authority.go

```diff
@@ -63,8 +63,7 @@ type ValidationAuthorityImpl struct {
 
 // NewValidationAuthorityImpl constructs a new VA
 func NewValidationAuthorityImpl(pc *cmd.PortConfig, sbc SafeBrowsing, caaClient caaPB.CAACheckerClient,
-	cdrClient *cdr.CAADistributedResolver, stats statsd.Statter, clk clock.Clock) *ValidationAuthorityImpl {
-	logger := blog.Get()
+	cdrClient *cdr.CAADistributedResolver, stats statsd.Statter, clk clock.Clock, logger blog.Logger) *ValidationAuthorityImpl {
```

Comment:
There seems to be some inconsistency in how functions with many arguments are defined: in one long line, or broken up into several indented lines.

A quick survey shows the expanded form being used prior to this PR for `NewCertificateAuthorityImpl`. Your PR converts `NewAmqpRPCServer` and `NewWebFrontEndImpl` to this form but leaves the `NewValidationAuthorityImpl` constructor as one long line. Similarly, `NewRegistrationAuthorityImpl` (not touched in this PR) remains one long line.

I propose that we leave `NewRegistrationAuthorityImpl` as is (since its outside the scope of this PR), but change `NewValidationAuthorityImpl` to the long form so that all of the parameter-heavy functions in this PR are defined consistently.

CC @jsha @rolandshoemaker since this is a stylistic matter. I couldn't find hard guidance in our own docs or in the broader CodeReviewComments doc. What do you folks prefer to see?